### PR TITLE
Ticket 23 configure bluesky logging

### PR DESCRIPTION
**Pull request by Chsudeepta**
### Description of work

Some system tests have been added for testing the bluesky logger

### Ticket

[*Link to Ticket*](https://github.com/ISISComputingGroup/ibex_bluesky_core/issues/23)

### Acceptance criteria

The system tests should run successfully as a complete set or when executed individually

### System tests

Added tests to check the following: 
1. Log folder is created when logging is requested
2. Log file is created when logging is requested
3. The log message is present in the log file.

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?
- [ ] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev)?

### Final Steps
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

**Review by jackbdoughty — CHANGES_REQUESTED** [review]
After these changes, running this test module in one go I get test failures on lines `224` and `212` of `assert os.path.exists(qualified_log_filename) == True` `AssertionError`. I defined `LOG_FILE_NAME = "bluesky.log"`

File: test_bluesky.py [hunk 1]
```diff
@@ -181,6 +187,45 @@ def _plan():
         # Assert we successfully set npoints periods
         self.assertEqual(g.get_number_periods(), npoints)
 
+    def test_GIVEN_logging_is_requested_THEN_the_log_folder_exists():
```
[the remark this answers is not in the document]
Comment (Chsudeepta) [reply test_bluesky.py:190]: Done.

File: test_bluesky.py [hunk 2]
```diff
@@ -181,6 +187,45 @@ def _plan():
...
+    def test_GIVEN_logging_is_requested_THEN_the_log_file_exists():
+        log_path = LOG_FOLDER
+        if LOG_ENV_PATH in os.environ:
+            log_path = os.environ[LOG_ENV_PATH]
+
+        # Log invocation.
+        this_function_name = sys._getframe().f_code.co_name
+        message = LOG_MESSAGE + this_function_name
+        logger.blueskylogger.info(message)
+        qualified_log_filename = os.path.join(log_path, LOG_FILE_NAME)
```
[the remark this answers is not in the document]
Comment (Chsudeepta) [reply test_bluesky.py:210]: Added missed constant

**Comment by Chsudeepta** [comment]
@jackbdoughty , I have corrected the issues, including the ruff ones

**Review by jackbdoughty — APPROVED** [review]

**Pull request merged**